src/MxpTag.h

```diff
@@ -33,7 +33,7 @@ class MxpTagAttribute : public QPair<QString, QString>
 
     MxpTagAttribute(const QString& name, const QString& value) : QPair(name, value) {}
 
-    explicit MxpTagAttribute(const QString& name) : MxpTagAttribute(name, QString::fromLatin1("")) {}
+    explicit MxpTagAttribute(const QString& name) : MxpTagAttribute(name, QStringLiteral("")) {}
```

Comment:
Actually the original is wrong - and should be changed to `QString()`. Only the `QString::isNull()` method can tell the difference between an empty `QString` and a null (default constructed) `QString`...